doc/en/user/source/data/webadmin/layers.rst

```diff
@@ -218,6 +218,20 @@ Cache parameters that apply to the HTTP response from client requests.
 
 * **Response Cache Headers**— If selected, GeoServer will not request the same tile twice within the time specified in :guilabel:`Cache Time`. One hour measured in seconds (3600), is the default value for :guilabel:`Cache Time`.
 
+Root Layer in Capabilities
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When a layer is the only top level element in the Capabilities document, it is possible to remove the Root Layer that contains it, and return
+a hierarchy where the layer is the root instead.
+
+To enable this functionality, choose the **No** option from the Root Layer in Capabilities section.
+By default this behaviour is inherited from the global WMS service settings (**WMS Global Settings** option).
+Finally, it is possible to override the service settings and force a **Yes** to always include the GeoServer root element.
```

Comment:
This may be easier to understand with a check box (on by default) to enable root layer, and when checked the fields to fill in description are enabled. 

"to enable select *no*" is the problem here, expressed as a double negative.

Can we express it as a positive and show what the effect of enable / disable will be?

Reply:
@jodygarnett I agree that inverting the logic is probably better, I will try to do that (also the checkbox is a good idea).
@aaime the 3 levels of configuration mimick what we already do for interpolation settings, where we can have a WMS service default, a layer layer override and a request level override.
They work in this order: WMS config is the default, layer (or group) config overrides the default if set to Yes or No, request parameter, if present,  wins over both.
I will try to improve documentation too.